stdlib/runtime/dataStructures.gr

```diff
@@ -55,6 +55,22 @@ provide primitive allocateInt32: () -> WasmI32 = "@allocate.int32"
 @unsafe
 provide primitive newInt32: WasmI32 -> WasmI32 = "@new.int32"
 
+/**
+ * Allocates a new Uint32.
+ *
+ * @returns The pointer to the empty Uint32
+ */
+@unsafe
+provide primitive allocateUint32: () -> WasmI32 = "@allocate.uint32"
+
+/**
+ * Allocates a new Uint32 with a prepopulated value
+ * @param value: The value to store
```

Comment:
```suggestion
 * Allocates a new Uint32 with a prepopulated value.
 *
 * @param value: The value to store
```

Reply:
There are also a few others in this file I can fix